audb/core/dependencies.py

```diff
@@ -359,13 +359,10 @@ def version(self, file: str) -> str:
         """
         return self._df.version[file]
 
-    def _add_media(
+    def _add_or_update_media(
```

Comment:
We should probably split into two functions - one for adding and one for updating media. Then we don't have to figure this out later on. But again, I would apply those changes step by step to see the benefit we gain by each of them.

Reply:
But we don't have to tackle it here. If I understand you correctly you propose anyway to first merge your changes and afterwards see if we can improve further in splitting them up, correct?

Reply:
No, I propose to do it in an separate PR.

Reply:
OK, then we handle adding a few more comments to `publish.py` there as well.